plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/sidebar-navigation-screen-typography.tsx

```diff
@@ -1,17 +1,36 @@
+/* eslint-disable @woocommerce/dependency-group */
+/* eslint-disable @typescript-eslint/ban-ts-comment */
 /**
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
 import { createInterpolateElement } from '@wordpress/element';
 import { Link } from '@woocommerce/components';
+import { useSelect } from '@wordpress/data';
+// @ts-ignore no types exist yet.
+import { BlockEditorProvider } from '@wordpress/block-editor';
+import { noop } from 'lodash';
```

Comment:
Are we still using this function from lodash? A while ago there was mention of reducing use of lodash to simplify future efforts to one day remove it. If this is the norm thats ok too

Reply:
Yes, we're still using it. 

> A while ago there was mention of reducing use of lodash to simplify future efforts to one day remove it.

Good point! However, I think we're using the external/global lodash via [dependency-extraction-webpack-plugin](https://github.com/WordPress/gutenberg/blob/trunk/packages/dependency-extraction-webpack-plugin/lib/util.js). It should be loaded whether we use it or not since `@wordpress/*` packages still depends on lodash.